dlk/python/dlk/templates/src/func/quantized_conv2d.cpp

```diff
@@ -167,19 +286,47 @@ void func_QuantizedConv2DWithThreshold(QUANTIZED_NOT_PACKED input[],
                        p.normal_conv_params.output_channels);
 }
 
-void func_QuantizedConv2DWithThreshold(QUANTIZED_NOT_PACKED input[],
+void func_QuantizedConv2DWithThreshold(QUANTIZED_PACKED input[],
                                        QUANTIZED_PACKED_KERNEL kernel[],
-                                       QUANTIZED_NOT_PACKED output[],
+                                       QUANTIZED_PACKED output[],
                                        T_FLOAT scaling_factor[],
                                        binary_convolution_parameters p) {
   func_QuantizedConv2DWithThreshold(input, kernel, output, scaling_factor[0],
                                     p);
 }
 
-void func_QuantizedConv2DWithThreshold(QUANTIZED_NOT_PACKED input[],
+void func_QuantizedConv2DWithThreshold(QUANTIZED_PACKED input[],
                                        QUANTIZED_PACKED_KERNEL kernel[], T_FLOAT output[],
                                        T_FLOAT scaling_factor[],
                                        binary_convolution_parameters p) {
   func_QuantizedConv2DWithThreshold(input, kernel, output, scaling_factor[0],
                                     p);
 }
+
+
+void func_Lookup(float *input, QUANTIZED_PACKED_KERNEL *lsb, QUANTIZED_PACKED_KERNEL *msb, QUANTIZED_PACKED* output, int h, int w, int c) {
+
+  int b = 32;
+  int packed_depth = 2;
```

Comment:
What is this?

Reply:
This is a hard coded part. The input image contains 3 channels, and, after the lookup, the depth will fit in 2 words of 32 bits each (total 30 channels). So:
  - Input depth becomes 30
  - Input (packed depth) becomes 2

Reply:
hardcoded constant should be constexpr value, and should be commented like `// hard coded, not configurable`

Reply:
Ah, what I mean is that shouldn't be hard coded in the first place.

Reply:
I mean, it is actually configurable. It could change.

Reply:
When you make it configurable? In this PR or another PR?

Reply:
Maybe it is better to do it later, there are already a lot of changes.
Put everything together and make it work first is maybe easier.

Reply:
OK.